specs/engage-digital_openapi3.yaml

```diff
@@ -1343,9 +1343,9 @@ paths:
           schema:
             type: string
         - description: >-
-            For an email or SMS source. For an email, an array containing the
+            For an email or SMS and WhatsApp source. For an email, an array containing the
             email addresses used in the “To” section of the email. This
-            parameter is mandatory when initiating a discussion. For a SMS, the
+            parameter is mandatory when initiating a discussion. For a WhatsApp message or a SMS, the
             number the SMS will be sent to. It must start with 00 or +, example:

```

Comment:
did you try with a phone number starting with 00 on whatsapp? I guess `PhoneUtils` will take care of it but better be safe than sorry

Reply:
Yep I tried, it works